client/src/app/site/pages/meetings/modules/poll/base/base-poll-detail.component.ts

```diff
@@ -276,6 +285,48 @@ export abstract class BasePollDetailComponent<V extends PollContentObject, S ext
         );
     }
 
+    private setLiveRegisterData(): void {
+        const userIds = new Set<Id>([]);
+        for (const group of this.poll.entitled_groups) {
+            const meetingUserIds = group.meeting_user_ids ?? [];
+            meetingUserIds.forEach(mu_id => userIds.add(this.meetingUserRepo.getViewModel(mu_id)?.user_id));
+        }
+        const delegates = new Set<Id>([]);
+        Array.from(userIds).forEach(userId => {
+            if (this.userRepo.getViewModel(userId)?.vote_delegated_to_id()) {
+                delegates.add(this.userRepo.getViewModel(userId)?.vote_delegated_to_id());
+            }
+        });
+        userIds.update(delegates);
+
+        this.subscriptions.push(
+            this.userRepo
+                .getViewModelListObservable()
+                .pipe(
+                    filter(users => !!users.length),
+                    map(users => users.filter(user => userIds.has(user.id)))
+                )
+                .subscribe(users => {
+                    const entries: EntitledUsersTableEntry[] = [];
+                    for (const user of users || []) {
+                        const delegateToId = user.vote_delegated_to_id();
+                        entries.push({
+                            id: user.id,
+                            user: user,
+                            voted_verbose: `voted:true`,
```

Comment:
Is this supposed to always be `voted:true`?
If no, fix it. If yes, you may as well remove this property.

Reply:
Fixed.